playstats/appstat.py

```diff
@@ -111,3 +112,27 @@ def title(self):
         selector = CSSSelector('.id-app-title')
         match = self.tree.xpath(selector.path)
         return match[0].text
+        
+    def keyword_rank(self, search):
+        # Returns the rank of the app on searching the passed keyword.
+        # Useful for playstore search optimisation.
+        # The rank is based on the search in the playstore website.
+        # Rank maybe off by 1 or 2 due to playstore advertisement.
+        # 0 means search didn't contain the app for the first 100 results
+        search_string = search.replace(' ', '%20')
+        url = 'https://play.google.com/store/search?q=' + search_string + '&c=apps&hl=en'
+        result = urlopen(url).read()
+        result_tree = html.fromstring(result)
+        match = result_tree.xpath('//@data-docid')
+        i = 1
+        rank = 1
+        try:
+            while(i < 500):
```

Comment:
Why not keep it as i<=100 and increment i by 1?

Reply:
The docid set has the same docid reoccurring 5 times.. that's why incrementing the count by 5. Multiplying was somehow offsetting the result. Will look into this though.

Reply:
Okay.